app/pages/index.tsx

```diff
@@ -1,39 +1,36 @@
 import { Suspense } from "react";
-import { BlitzPage, useMutation } from "blitz";
+import { BlitzPage, useMutation, useRouter } from "blitz";
 import Layout from "app/core/layouts/Layout";
 import { useCurrentUser } from "app/core/hooks/useCurrentUser";
 import logout from "app/auth/mutations/logout";
 import { Button } from "@ableco/abledev-components";
+import protectPage from "app/core/helpers/protectPage";
 
 const Home: BlitzPage = () => {
   const currentUser = useCurrentUser();
+  const router = useRouter();
   const [logoutMutation] = useMutation(logout);
 
   return (
     <div>
-      {currentUser ? (
-        <div>
-          {currentUser.picture ? (
-            <img
-              src={currentUser.picture}
-              alt="Profile picture"
-              width="96"
-              height="96"
-            />
-          ) : null}
-          <Button
-            variant="primary"
-            size="base"
-            onClick={() => logoutMutation()}
-          >
-            Logout
-          </Button>
-        </div>
-      ) : (
-        <Button variant="primary" size="base">
-          <a href="/api/auth/google">Log In</a>
-        </Button>
-      )}
+      {currentUser.picture ? (
+        <img
+          src={currentUser.picture}
+          alt="Profile picture"
+          width="96"
+          height="96"
+        />
+      ) : null}
+      <Button
+        variant="primary"
+        size="base"
```

Comment:
I wonder if can enforce this as defaults 🤔

Reply:
I was thinking the same, but I'm not 100% sure of what the defaults should be. I barely added defaults because of that.

Reply:
Anyway, I think it was safe in this case, so I added some defaults on the library and updated them here.